desafio-02/lestterx/javascript/README.md

```diff
@@ -0,0 +1 @@
+### DESAFIO 02 - Primos
```

Comment:
@LestterX pode incrementar o teu README com algumas informações?

Qual linha de raciocínio utilizou para fazer teu programa, descreva aqui.

E como executou? Foi pelo Node no terminal? Usou alguma outra coisa? Quem for executar teu código deve seguir quais passos?